hazelcast/src/main/java/com/hazelcast/nio/MemberSocketInterceptor.java

```diff
@@ -22,12 +22,16 @@
 /**
  * Member Socket Interceptor can be registered via
  * see {@link com.hazelcast.config.SocketInterceptorConfig}
+ *
+ * Warning: a MemberSocketInterceptor provides access to the socket and will bypass
+ * any TLS encryption. So be warned that any data send using the SocketInterceptor
+ * could be visible as plain text and could therefor be a huge security risk.
```

Comment:
I would avoid the `huge`.

Reply:
enormous? :)

Reply:
just "a security risk", let users decide impact themselves